frontend/public/components/dashboard/dashboards-page/cluster-dashboard/health-item.tsx

```diff
@@ -235,7 +239,7 @@ export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsyste
   const healthState = healthHandler(resourcesResult);
 
   const PopupComponentCallback = React.useCallback(
-    () => <AsyncComponent loader={popupComponent} resourcesResult={resourcesResult} />,
+    () => <AsyncComponent loader={popupComponent} resources={resourcesResult} />,
```

Comment:
I would prefer doing
```suggestion
    () => <AsyncComponent loader={popupComponent} {...resourcesResult} />,
```